src/main/java/org/mskcc/cbio/oncokb/web/rest/CronJobController.java

```diff
@@ -261,7 +262,21 @@ public void checkExposedTokens() throws IOException, InterruptedException {
                 } 
             }         
 
-            if (githubCount > 0 || googleCount > 0 || baiduCount > 0){   
+            if (githubCount > 0){   
+                ExposedToken t = new ExposedToken();
+                UserDTO user = userMapper.userToUserDTO(token.getUser());
+                t.setToken(q);
+                t.setEmail(user.getEmail());
+                t.setFirstName(user.getFirstName());
+                t.setLastName(user.getLastName());
+                t.setLicenseType(user.getLicenseType() != null ? user.getLicenseType().name() : "");
+                t.setSource("GitHub");   
```

Comment:
I believe this part of the logics are duplication of the below. It would be better to unify.